src/generateRandomNumber.js

```diff
@@ -0,0 +1,9 @@
+export const generateRandomNumber = () => {
+  let res = new Set([]);
+
+  while (res.size !== 4) {
+    res.add(Math.floor(Math.random() * 9) + 1);
+  }
+
+  return [...res].join('');
```

Comment:
It's better to return an integer or an array of digits for better manipulation instead of a string.
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/70009205/8d318016-1fc9-4bea-8025-a594b0886908)